org.eclipse.scanning.api/src/org/eclipse/scanning/api/malcolm/connector/MalcolmMethod.java

```diff
@@ -0,0 +1,29 @@
+package org.eclipse.scanning.api.malcolm.connector;
+
+import org.eclipse.scanning.api.malcolm.message.MalcolmMessage;
+
+/**
+ * An enumeration of the malcolm methods that can be set in a {@link MalcolmMessage}. 
+ * 
+ * @author Matthew Dickie
+ */
+public enum MalcolmMethod {
+	
+	ABORT,
+	CONFIGURE,
+	DISABLE,
+	PAUSE,
+	RESET,
+	RESUME,
+	RUN,
+	VALIDATE;
+	
+	/**
+	 * Returns the name of this message in lower case, as expected by the malcolm device
+	 * over a communication channel.
+	 * @see java.lang.Enum#toString()
+	 */
+	public String toString() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1161)